src/descriptors-device.adb

```diff
@@ -43,6 +43,27 @@ package body Descriptors.Device is
       Output_Dir : String);
    --  Dump the IRQ names and trap handlers
 
+   -----------------
+   -- MCU_To_CPU --
+   -----------------
+
+   function MCU_To_CPU
+     (Device : Device_T) return Unbounded_String
+   is
+      Descrition_String : constant String := To_String (Device.Short_Desc);
+      CPU : Unbounded_String;
+   begin
+      if Starts_With (S1 => Descrition_String, S2 => "STM32F0") then
```

Comment:
This patch is very useful, but I'm a bit uncomfortable with it: the <cpu> element is supposed to be there for this purpose, while here we skip it entirely. So this makes svd2ada very dependant on ST-generated SVD files (and they don't seem to generate this optional information, while they should), while other vendors do and so we should first check if the information is present there.

Basically, I'm not refusing such consideration, it's only that to be fair, the SVD standard already provides means to provide this information, and having such routine that is ST-specific there without the more generic handling implemented first seems unfair, and risky (if ST decides at some point to fill the gaps).

Reply:
I agree that's cause for concern.